_docs/platform.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Simpl Platform
+permalink: /docs/platform/
+layout: docs
+description: Simpl Platform
+---
+
+## Simpl Platform
+
+  <div class="container about">
+    <div class="row">
+      <div class="col-sm-12">
+      <h5 class="u-title u-center u-spacerUp">Standard Simpl Framework</h5>
+        <p class="text-center">The simpl-games-api is required to run one or more games. For each game, a model service and user interface (UI) will need to be created.
+        </p>
+      <section>
+        <div class="row">
+          <div class="col-sm-4 u-center wow slideInUp" data-wow-offset="140">
+            <div class="circle">
+              <svg fill="#FFFFFF" id="gift" height="30" viewBox="0 0 24 24" width="30" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M17 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V7l-4-4zm-5 16c-1.66 0-3-1.34-3-3s1.34-3 3-3 3 1.34 3 3-1.34 3-3 3zm3-10H5V5h10v4z"/>
+                  <path d="M0 0h24v24H0z" fill="none"/>
+              </svg>
+            </div>
+            <h5>simpl-games-api</h5>
+            <p class="tech__points--para">
+              A single instance of simpl-games-api provides a database and REST API endpoints for one more more games. When data is changed, an event is sent to the registered game model service via an HTTP Webhook.
+            </p>
+          </div>
+          <div class="col-sm-4 u-center wow slideInUp" data-wow-offset="140">
+            <div class="circle">
+              <svg fill="#FFFFFF" id="gift" height="30" viewBox="0 0 24 24" width="30" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zM6.5 9L10 5.5 13.5 9H11v4H9V9H6.5zm11 6L14 18.5 10.5 15H13v-4h2v4h2.5z"/>
+                  <path d="M0 0h24v24H0z" fill="none"/>
+              </svg>
+            </div>
+            <h5>Game Model Service</h5>
+            <p class="tech__points--para">
+              For each game, a model service must be created that provides communication between the simpl-games-api and UI.
+            </p>
+          </div>
+          <div class="col-sm-4 u-center wow slideInUp" data-wow-offset="140">
+            <div class="circle">
+              <svg fill="#FFFFFF" id="gift" height="30" viewBox="0 0 24 24" width="30" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M22 18V3H2v15H0v2h24v-2h-2zm-8 0h-4v-1h4v1zm6-3H4V5h16v10z"/>
+                  <path d="M0 0h24v24H0z" fill="none"/>
+              </svg>
+            </div>
+            <h5>Game UI</h5>
+            <p class="tech__points--para">
+              For each game, a browser-based user UI must be created that communicates with the game model service. We recommend using simpl-ui-cookiecutter and the simpl-react package that comes pre-installed in it.
+            </p>
```

Comment:
simpl-react does not come pre-installed in simpl-ui-cookiecutter.  All game ui's require simpl-react whether or not they are created using simpl-ui-cookiecutter

Reply:
I put that because it's included in the requirements file. I will removed the pre-the installed part.